src/nnet3/nnet-training.h

```diff
@@ -93,6 +93,10 @@ struct NnetTrainerOptions {
 // Also see struct AccuracyInfo, in nnet-diagnostics.h.
 struct ObjectiveFunctionInfo {
   int32 current_phase;
+  int32 minibatches_this_phase; // number of minibatches processed in this phase
+                                // to compute this objective function.
```

Comment:
change comment to:
`The number of minibatches' worth of stats that we accumulated in the
  phase numbered 'current_phase'.`